.evergreen.yml

```diff
@@ -102,16 +102,6 @@ functions:
         shell: bash
         script: |
           source ~/compass_env.sh
-
-          echo "Configuring npm..."
-          cat <<EOT >> .npmrc
-          registry=https://registry.npmjs.org/
-          devdir=$NPM_CACHE_DIR/.node-gyp
-          init-module=$NPM_CACHE_DIR/.npm-init.js
-          cache=$NPM_CACHE_DIR
-          tmp=$NPM_TMP_DIR
```

Comment:
I know that we just cleaned those up, but I'm thinking that macos might be failing because we are running npm7 with global temp and cache folders in the config and maybe this somehow messes it up (if it doesn't help we can try adding `--force` flag in `npm install` in `hadron-build` as a last resort)